### ISSUE TITLE
Bulk update with new api

### ISSUE THREAD

**Issue by MichaelSowah** [tekartik/sembast.dart#94]
How do I do bulk update with new api since updateRecords is dprecated?

**Comment by alextekartik** [comment]
You can upsert multiple records using a [RecordsRef](https://pub.dev/documentation/sembast/latest/sembast/RecordsRef-class.html)
```dart
// Upsert multiple records
var records = await store.records([key1, key2]);
var result = await records.put(
  db,
  [
    {'value': 'new value for key1'},
    {'value_other': 'new value for key2'}
  ],
  merge: true);
```

You can also update multiple records with the same values on a given store [using a finder](https://pub.dev/documentation/sembast/latest/sembast/StoreRef/update.html).

It seems that we are missing `RecordsRef.update` which I guess I did need so far. I'll look into it.

**Comment by MichaelSowah** [comment]
Thanks, just wanted to make sure< I have not missed anything. I guess I will just loop and update the data 1 after the other.
Can I do this eg: var records = store.records([key1, key2]);  but instead of keys I can pass say ids or another parameter or I have to use the finder to do that?

**Comment by alextekartik** [comment]
`RecordsRef.update` has been added in the latest published version 2.1.1. This is only for 'by  key'. Any other parameter should use a finder
